packages/@o3r/schematics/project.json

```diff
@@ -29,17 +29,7 @@
       "executor": "nx:run-script",
       "options": {
         "script": "build:builders"
-      },
-      "outputs": [
```

Comment:
` "{projectRoot}/dist/package.json",`
 should be added in nx.json for `prepare-build-builders`

Reply:
Not really
As mentionned by @kpanot , this is quite unusual the package.json is copied into the dist by the ama-sdk task.
This is an output that is specific by the way of this package is build, the prepare:build-builders do not touch the package.json at all.